app/src/main/java/com/github/featuredetectandroid/utils/SelectFeatureDetector.kt

```diff
@@ -0,0 +1,15 @@
+package com.github.featuredetectandroid.utils
+
+import android.content.Context
+import com.github.featuredetectlib.learned.SuperPoint
+
+// TODO: add other FeatureDetector's after their implementation
+fun selectFeatureDetector(
```

Comment:
I think, this doesn't need its own file. We can place it in `MainActivity` as it won't be too big even with a lot of detectors.

Reply:
Actually, read the next comment

Reply:
deleted